packages/lesswrong/lib/collections/conversations/fragments.ts

```diff
@@ -10,22 +26,46 @@ registerFragment(`
     participants {
       ...UsersMinimumInfo
     }
+    latestMessage {
+      ...messageListFragment
+    }
     archivedByIds
     messageCount
     moderator
   }
 `);
 
+/**
+ * @deprecated Use ConversationsMinimumInfo
+ * TODO Remove after 2023-11-03
+ * */
 registerFragment(`
-  fragment newConversationFragment on Conversation {
+  fragment conversationIdFragment on Conversation {
     _id
+  }
+`);
+
+registerFragment(`
+  fragment ConversationsMinimumInfo on Conversation {
```

Comment:
I just changes these to fit the usual naming convention, and also merged `conversationIdFragment` and `newConversationFragment` because they should be about equally costly to fetch